CONTRIBUTION.md

```diff
@@ -20,6 +20,19 @@ make test
 
 > Helmsman v1.x supports helm v2.x only and will no longer be supported except for bug fixes and minor changes.
 
+## Commit messages
+
+This project uses [Conventional Commits](https://www.conventionalcommits.org/) to auto-generate release notes.
+
+| Type | Purpose | Example |
+|------|---------|---------|
+| `feat!:` | Breaking change | `feat!: remove deprecated API` |
+| `feat:` | New feature | `feat: add parallel release execution` |
+| `fix:` | Bug fix | `fix: helm 3.x compatibility in Dockerfile` |
+| `chore:` | Build, CI, deps, docs, infra | `chore: update goreleaser config` |
```

Comment:
What about `doc` ?

Reply:
I tought of it to fall into 'chore' as well like chore(deps), chore(docs) or just chore